packages/frontend/components/item/item-info-auction/component.tsx

```diff
@@ -66,7 +66,7 @@ export const ItemInfoAuction = ({
   }, [item.id, user, dispatch]);
 
   const targetDate = new Date(item.endDate);
-  const minBidAmount = +item.currentPrice + +item.minimalBid + 1;
+  const minBidAmount = +item.currentPrice + +item.minimalBid;
```

Comment:
i think you dont need  "+price" more, because i refactor productDto and now prices have number type